app/build.gradle

```diff
@@ -26,18 +27,33 @@ android {
     dataBinding{
```

Comment:
Yes, this is the updated one but I think it can only be added from Android Studio version 4.1+. I don't think there will be any problem using the existing implementation.

Reply:
Okay 👍